straxen/plugins/peaks/peaks_som.py

```diff
@@ -0,0 +1,81 @@
+import strax
+import numpy as np
+from straxen.plugins.peaks.peaks import Peaks
+
+export, __all__ = strax.exporter()
+
+
+@export
+class PeaksSOM(Peaks):
```

Comment:
This plugin is used in the SOM context to propagate the SOM parameters from `peaklet_classification` to `peaks`.